modules/dnn/src/webgpu/src/op_softmax.cpp

```diff
@@ -78,17 +78,15 @@ bool OpSoftmax::forward(Tensor& in, Tensor& out)
     if (max_tensor_ == NULL || sum_tensor_ == NULL)
     {
         std::vector<int> shape = {outer_size_, channel_size_};
-        max_tensor_ = new Tensor(NULL, shape, wgpu::BufferUsage::Storage | wgpu::BufferUsage::CopySrc);
-        sum_tensor_ = new Tensor(NULL, shape, wgpu::BufferUsage::Storage | wgpu::BufferUsage::CopySrc);
+        max_tensor_ = new Tensor(NULL, shape);
+        sum_tensor_ = new Tensor(NULL, shape);
     }
     if(uniformTensor_ == NULL && needsUniform) 
     {
         SoftmaxParam param = {channel_size_, outer_size_, channels_, 
                               log_softmax_ == true ? 1 : 0};
         uniformTensor_ = new Tensor((const void*) &param, 
-                                    sizeof(SoftmaxParam), 
-                                    wgpu::BufferUsage::Uniform | wgpu::BufferUsage::CopyDst, 
-                                    wFormatInt32);
+                                    sizeof(SoftmaxParam));
```

Comment:
Oh, I get your reason why to specify usage flag in the constructor's argument.

However, Tensor is a concept on the context of dnn network or more generally, "computation graph". The uniform buffer is an optimization method specific to a library. e.g. webgpu. I suggest to use webgpu::Buffer object directly for uniform parameters in your operator's implementation. So that we can simply keep the Tensor a read/write ndarray.

Reply:
Thanks, can I use different Tensor constructors to omit buffer usage?

Reply:
No need to add usage parameter for Tensor constructor, since Tensor object's usage is always "wgpu::BufferUsage::Storage | wgpu::BufferUsage::CopyDst | wgpu::BufferUsage::CopySrc"

Reply:
Made corresponding changes, PTAL. Sorry for the delay. @wzw-intel